modules/KalturaSupport/resources/mw.KBaseMediaList.js

```diff
@@ -268,7 +268,7 @@
 		//Media Item
 		renderMediaList: function(callback){
 			//Only render if medialist item are present
-			if (this.getTemplateData().length > 0) {
+			if (this.getTemplateData().length > (0 - !this.getConfig('showEmptyPlaylistError'))) {
 				//Generate new list template data
```

Comment:
this code is needed because we need to render the template even if the length of media items is zero, because the empty playlist error is handled inside the template